charts/seqr/values.yaml

```diff
@@ -156,18 +156,10 @@ volumeMounts: {}
 
 jobAfterHook: ""
 jobBeforeHook: ""
-cronJobs: {}
-# Examples
-# cronJobs:
-#  - name: sleep
-#    schedule: "5-59/5 * * * *"
-#    command: "/bin/sleep 2"
-#    timeout_s: 3600
-#  - name: fail_50
-#    schedule: "6-59/5 * * * *"
-#    command: "/bin/sleep 2 && exit $((RANDOM % 2))"
-
-checkForNewSamplesJobEnabled: false
+cronJobs:
+  - name: check-new-samples-job
+    schedule: "*/10 * * * *"
```

Comment:
I don't think this is the right cron expression - isn't the first value seconds?

Reply:
From [Crontab.guru](https://crontab.guru/#*/10_*_*_*_*), this should be every 10th minute.

Reply:
yeah but that website does not appear to have seconds in its cron expression but I think our cron evaluator does...

Reply:
Maybe the old cron, but [Kubernetes crons are minute first](https://kubernetes.io/docs/concepts/workloads/controllers/cron-jobs/#schedule-syntax)